src/invidious/videos.cr

```diff
@@ -818,7 +818,7 @@ end
 
 def extract_polymer_config(body)
   params = {} of String => JSON::Any
-  player_response = body.match(/(window\["ytInitialPlayerResponse"\]|var\sytInitialPlayerResponse)\s*=\s*(?<info>{.*?});/m)
+  player_response = body.match(/(window\["ytInitialPlayerResponse"\]|var\sytInitialPlayerResponse)\s*=\s*(?<info>{.*?});\s*var\s*meta/m)
```

Comment:
I have noticed that this extra javascript (`var meta = ...`) is not always present in the script.
I can't tell what triggers it (I've tried multiple combinations of URL arguments, with no luck).

Reply:
Huh that's strange. Its always been present on my end.